crypto/rand/rand_lib.c

```diff
@@ -547,6 +541,13 @@ unsigned char *rand_pool_detach(RAND_POOL *pool)
     return ret;
 }
 
+/*
+ * Re-attach the |pool| buffer.
```

Comment:
The comment needs to make it more clear that it is only allowed to pass a |buffer| which was previously detached from the same pool.